src/packages/rte/tiptap/components/input-tiptap/input-tiptap.element.ts

```diff
@@ -120,13 +113,18 @@ export class UmbInputTiptapElement extends UmbFormControlMixin(UmbLitElement, un
 	}
 
 	override render() {
-		if (!this._extensions?.length) return html`<uui-loader></uui-loader>`;
 		return html`
-			<umb-tiptap-hover-menu .editor=${this._editor}></umb-tiptap-hover-menu>
-			<umb-tiptap-fixed-menu
-				?readonly=${this.readonly}
-				.editor=${this._editor}
-				.extensions=${this._extensions}></umb-tiptap-fixed-menu>
+			${when(
+				!this._editor && !this._extensions?.length,
+				() => html`<uui-loader></uui-loader>`,
+				() => html`
+					<umb-tiptap-hover-menu .editor=${this._editor}></umb-tiptap-hover-menu>
+					<umb-tiptap-fixed-menu
+						.editor=${this._editor}
+						.extensions=${this._extensions}
```

Comment:
This isn't a property on the fixed menu anymore, is it?

Reply:
Good spot, I've updated the PR. 👍